analysis/topeft_run2/make_cards.py

```diff
@@ -26,8 +26,8 @@
 log    = {condor_dir}/job_{idx}.log
 
 request_cpus = 1
-request_memory = 8192
-request_disk = 1024
+request_memory = 20000
+request_disk = 4096
```

Comment:
Does the datacard maker really need this much more memory?

Reply:
When also using the tau channels, yes